components/google_calendar/google_calendar.app.js

```diff
@@ -1,154 +1,307 @@
-const { google } = require("googleapis")
-const get = require("lodash.get")
+const { google } = require("googleapis");
+const get = require("lodash.get");
 
 module.exports = {
   type: "app",
   app: "google_calendar",
   propDefinitions: {
     calendarId: {
+      label: "Calender Name",
```

Comment:
Optional:

I'm getting these errros with eslint: **Component prop iCalUID must have a label. See [pipedream/props-label](https://pipedream.com/docs/components/guidelines/#labels)** 

But if you want you can add a label to these properties too:
```
iCalUID
maxAttendees
maxResults
orderBy
pageToken
privateExtendedProperty
q
sharedExtendedProperty
showDeleted
showHiddenInvitations
singleEvents
syncToken
timeMax
timeMin
updatedMin
```